lib/modules/dosomething/dosomething_campaign/dosomething_campaign.theme.inc

```diff
@@ -439,7 +439,8 @@ function dosomething_campaign_preprocess_action_page(&$vars, &$wrapper) {
   $base_url = url(base_path(), array('absolute'=> TRUE, 'language' => 'en-global'));
   $campaign_path = url(current_path(), array('absolute' => TRUE));
   $country_prefix_for_share = (dosomething_global_get_current_prefix()) ? strtoupper(dosomething_global_get_current_prefix()) : 'global';
-  if (dosomething_campaign_feature_on($campaign, 'enable_voter_registration', TRUE)) {
+  if (dosomething_campaign_feature_on($campaign, 'social_share_unique_link')) {
+      $vars['social_share_custom_description'] = $campaign->variables['social_share_custom_description'];
       $vars['share_image'] = dosomething_image_get_themed_image_url($campaign->variables['share_image_nid'], 'landscape');
   }
   else {
```

Comment:
I think we can safely remove the else here, since it seems like we don't want problem shares at all anymore.